include/experimental/__p1673_bits/blas1_dot.hpp

```diff
@@ -148,6 +148,20 @@ namespace dot_detail {
     std::experimental::mdspan<ElementType1, std::experimental::extents<ext1>, Layout1, Accessor1> x,
     std::experimental::mdspan<ElementType2, std::experimental::extents<ext2>, Layout2, Accessor2> y)
   -> decltype(x(0) * y(0));
+
+  template<
```

Comment:
Please see comment below -- thanks!

Reply:
you mean above about removing the dotc_return type?